src/devices/Ssd1331/Ssd1331/Samples/SSDTest/SSDTest.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>

```

Comment:
similar comment for this CS csproj, see: https://github.com/dotnet/iot/blob/main/src/devices/Ssd1351/samples/Ssd1351.Samples.csproj

And also name it the same way!